bedrock-devnet/devnet/__init__.py

```diff
@@ -242,8 +256,9 @@ def devnet_l1_genesis(paths):
     if paths.fork_public_network:
```

Comment:
I think we don't have to use anvil when we create l1_genesis. I think we can use the geth same as legacy because you developed fork-public-network.py!

Reply:
Yes, I agree! Let me update

Reply:
Updated at [a03fb73d5b8c87128c450d7f56db7abd732f50a9](https://github.com/tokamak-network/tokamak-titan-canyon/pull/87/commits/a03fb73d5b8c87128c450d7f56db7abd732f50a9)

Reply:
Thank you! Let me test it!